pkg/crc/config/callbacks.go

```diff
@@ -11,5 +13,5 @@ func RequiresRestartMsg(key string, _ interface{}) string {
 }
 
 func SuccessfullyApplied(key string, value interface{}) string {
-	return fmt.Sprintf("Successfully configured %s to %s", key, value)
+	return fmt.Sprintf("Successfully configured %s to %s", key, cast.ToString(value))
```

Comment:
@guillaumerose In the commit logs you mentioned remove `%v` but here we are already using `%s` and the value already casted to string?

Reply:
yes sorry I tried to say badly that I could either use cast.ToString or just use %v. I chose cast.ToString because we use it a lot in this package.

Reply:
added !